typescript/widgets/src/icons/Help.tsx

```diff
@@ -0,0 +1,42 @@
+import React, { memo } from 'react';
+
+import { ColorPalette } from '../color.js';
+import { IconButton } from '../components/IconButton.js';
+
+import { QuestionMarkIcon } from './QuestionMark.js';
+
+function _HelpIcon({
```

Comment:
This is redundant with the existing Tooltip component in this lib. Let's keep that other one and cut this one

Reply:
I removed this component but our `Tooltip` component has an anchor tag as the parent, not sure if that is something we should look into or it was intended to be that way